src/main/java/salesforce/ui/pages/lightning/campaign/CampaignCreatedPage.java

```diff
@@ -0,0 +1,92 @@
+package salesforce.ui.pages.lightning.campaign;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+/**
+ * This class for elements of Campaign page created.
+ */
+public class CampaignCreatedPage extends BasePage {
+
+    @FindBy(css = ".slds-theme--success")
+    private WebElement alertSuccess;
+
+    @FindBy(css = "[class='slds-media__body'] span[class='custom-truncate uiOutputText']")
+    private WebElement createdCampaingTitle;
+
+    @FindBy(xpath = "//ul[contains(@class,\'branding-actions\')]/li/div//a[@role=\'button\']")
+    private WebElement createdCampaignOptionBtn;
+
+    @FindBy(xpath = "//a[@data-tab-name='detailTab']")
+    private WebElement detailsTab;
+
+    @FindBy(xpath = "//div/div/span[text()=\"Campaign Name\"]/../..//span/span")
+    private WebElement campaignNameCreated;
+
+    @FindBy(xpath = "//span[text()='Created By']/../../div/span/span")
+    private WebElement createBy;
+
+
+    public MenuPage clickCreatedCampaignOptionBtn() {
+        createdCampaignOptionBtn.click();
+        return new MenuPage();
+    }
+
+    @Override
+    protected void waitForPageLoaded() {
+        wait.until(ExpectedConditions.visibilityOf(alertSuccess));
```

Comment:
You have to use webelemt action instead of using a wait directly

Reply:
ok, It was solved